default/python/AI/ProductionAI.py

```diff
@@ -1822,7 +1819,7 @@ def _may_enqueue_for_stability(building_type: BuildingType, new_turn_cost: float
 
 
 def _build_scanning_facility() -> float:
-    """Consider building Scanning Facilities"""
+    """Consider building Scanning Facilities, returns added turn costs"""
```

Comment:
https://peps.python.org/pep-0257/#one-line-docstrings
> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.

Also, both parts of the sentence should use the same style.

```suggestion
    """Consider building Scanning Facilities, return added turn costs"""
```